src/app/shared/components/filters-list/working-hours/working-hours.component.ts

```diff
@@ -55,18 +55,30 @@ export class WorkingHoursComponent implements OnInit, OnDestroy {
       takeUntil(this.destroy$),
       debounceTime(300),
       distinctUntilChanged(),
-    ).subscribe((time: string) => this.store.dispatch(new SetStartTime(time.split(':')[0])));
+    ).subscribe((time: string) => this.store.dispatch(new SetStartTime(time?.split(':')[0])));
```

Comment:
what the reason of this change?

Reply:
because of new functionality added for the reset input